src/screens/SSISummaryScreen/index.tsx

```diff
@@ -65,8 +66,8 @@ class SSIOnboardingSummaryScreen extends PureComponent<IProps> {
     ];
 
     return (
-      <Container>
-        <SSITabView routes={routes} />
+      <Container style={{backgroundColor:backgrounds.secondaryDark}}>
```

Comment:
Create a new styling component for this container specifically for this screen with the background color

Reply:
created a new component for `SSISummaryScreen` container